apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    IDetailQueryResult,
+    ITranscribedAudioViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Link } from 'react-router-dom';
+import { routes } from '../../../app/routes/routes';
+import './transcribed-audio-detail.thumbnail.presenter.css';
+
+export const TranscribedAudioDetailThumbnailPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const location = `/${routes.resources
```

Comment:
Don't save a reference to this. In line it below. Otherwise, come up with a more accurate name than `location`.

Reply:
This is the same issue as in the Photograph branch.  I just need to see what you have in mind by implicit return and I can update all of these.

Reply:
Corrected this.